File: src/LinksNews/angular2App/app/core/domain/language.ts
import * as _ from 'lodash';

export class Language {
    id: number;
    code: string;
    name: string;
    supportedByInterface: boolean;
    supportedByNews: boolean;

    clone(): Language {
        return <Language>_.assign(new Language(), this);
    }

    static from (language: Language): Language {
        let result: Language = new Language();
        _.assign(result, language);
        return result;
    }
}

File: src/LinksNews/Services/Data/Models/Language.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;

namespace LinksNews.Services.Data.Models
{
    [Table("language")]
    public class Language
    {
        public long? Id { get; set; }
        public string Code { get; set; }
        public string Name { get; set; }
        public bool SupportedByInterface { get; set; }
        public bool SupportedByNews { get; set; }
    }
}
